tt_metal/llrt/llrt.cpp

```diff
@@ -124,11 +124,6 @@ void write_launch_msg_to_core(chip_id_t chip, const CoreCoord core, launch_msg_t
     }
 }
 
-static const uint32_t ERISC_APP_FW_ON_CORE_FLAG = 0x1;
-void launch_erisc_app_fw_on_core(chip_id_t chip, CoreCoord core) {
-    llrt::write_hex_vec_to_core(chip, core, tt::stl::Span(&ERISC_APP_FW_ON_CORE_FLAG, 1), eth_l1_mem::address_map::LAUNCH_ERISC_APP_FLAG);
-}
-
```

Comment:
beautiful!

Reply:
When you are ready to merge this, I believe you can also rm -rf:
`#include "eth_l1_address_map.h" // address_map`